src/main/java/com/uvocab/api/service/AuthService.java

```diff
@@ -0,0 +1,27 @@
+package com.uvocab.api.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import uvocab.protobuf.v1.User;
+
+@Service
+@AllArgsConstructor
+public class AuthService {
+
+  private final AuthenticationManager authenticationManager;
+
+  private static final String FIXED_TOKEN = "str_token";
+
+  public String login(User user) {
+    Authentication authentication =
+        authenticationManager.authenticate(
+            new UsernamePasswordAuthenticationToken(user.getLogin(), user.getPassword()));
+    SecurityContextHolder.getContext().setAuthentication(authentication);
```

Comment:
@Danilosimoes we don't need to do that, spring will do it for us "behind the scenes"